setup.cfg

```
@@ -30,6 +30,7 @@ packages = find:
 install_requires =
     audeer >=1.20.0
     dohq-artifactory >=0.8.1
+    pywin32 ;platform_system=='Windows'
```

In https://setuptools.pypa.io/en/latest/userguide/dependency_management.html they use

```suggestion
    pywin32; platform_system=='Windows'
```